core/src/main/java/org/owasp/dependencycheck/data/update/nvd/NvdCveParser.java

```diff
@@ -117,6 +125,24 @@ public void parse(File file) throws UpdateException, CorruptedDatastreamExceptio
         }
     }
 
+    /**
+     * Returns the Java major version as a whole number.
+     *
+     * @return the Java major version as a whole number
+     */
+    private static int getJavaVersion() {
+        String version = System.getProperty("java.version");
```

Comment:
```suggestion
        String version = System.getProperty("java.specification.version");
```

I think it will be better to use the java.specification.version


So that we don't have to change the logic to support the early-access releasese <major>-ea (see [JEP-223](https://openjdk.org/jeps/223))